.github/workflows/openms_clang_format.yml

```diff
@@ -29,5 +29,5 @@ jobs:
       if: steps.changed_files.outputs.any_modified == 'true'
       uses: DoozyX/clang-format-lint-action@v0.13
       with:
-        source: ${{ steps.changed_files.outputs.all_modified_files }}
+        source: ${{ steps.changed_files.outputs.all_changed_files }}
```

Comment:
Does that include added files? I'm never sure.

Reply:
Yes. This is not clear from their names (they should rethink all the names) `all_modified_files` is "ACMRD" and `all_changed_files` is "ACMR"

A: Added
C: Copied
M: Modified
R: Renamed
D: Deleted

Reference: https://github.com/tj-actions/changed-files

Reply:
Perfect, thank you.